testutils/channel_opts.go

```diff
@@ -124,10 +127,16 @@ func (o *ChannelOpts) DisableLogVerification() *ChannelOpts {
 
 // AddLogFilter sets an allowed filter for warning/error logs and sets
 // the maximum number of times that log can occur.
-func (o *ChannelOpts) AddLogFilter(filter string, maxCount uint) *ChannelOpts {
+func (o *ChannelOpts) AddLogFilter(filter string, maxCount uint, fields ...string) *ChannelOpts {
```

Comment:
Yeah neither. I really wanted to use varargs, and this seems to be the easiest way to avoid verbose call sites. Since it's only a test utility and not part of a public API, I left it in.